deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -71,8 +71,10 @@ class Agent:
         short_version (str): Agent version in format x.y
         cypher (str): Encryption method for message communication.
         os (str): Agent operating system.
-        fim_eps (int): Set the maximum event reporting throughput. Events are messages that will produce an alert.
-        fim_integrity_eps (int): Set the maximum database synchronization message throughput.
+        fim_eps (int): Set the fim's maximum event reporting throughput.
+        fim_integrity_eps (int): Set the fim_integrity's maximum event reporting throughput.
+        syscollector_eps (int): Set the syscollector's maximum event reporting throughput.
+        rootcheck_eps (int): Set the rootcheck's maximum event reporting throughput.
```

Comment:
Add default values to the documentation

```suggestion
        fim_eps (int): Fim's maximum event reporting throughput. Default `1000`.
        fim_integrity_eps (int): Fim integrity's maximum event reporting throughput. Default `100`.
        syscollector_eps (int): Syscollector's maximum event reporting throughput. Default `100`.
        rootcheck_eps (float): Rootcheck's maximum event reporting throughput. Default `60.0`.
```